Introduce support for the path operators `{*}` and `{**}` (#1411)
kyma-project/api-gateway: docs/user/custom-resources/apirule/v2alpha1/04-10-apirule-custom-resource.md

| **rules.service**                                |  **NO**  | Services definitions at this level have higher precedence than the Service definition at the **spec.service** level.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | None                                                                                                                  |

> | **rules.path**                                   | **YES**  | Specifies the path on which the service is exposed. Supported configurations are: Exact path (e.g. `/abc`) - matches the specified path exactly; Usage of the `{*}` operator (e.g. `/foo/{*}` or `/foo/{*}/bar`) - match any request that matches the pattern with exactly one path segment in the operator's place; Usage of the `{**}` operator (e.g. `/foo/{**}` or `/foo/{**}/bar`), `{**}` must be used as the last operator in the path - match any request that matches the pattern with zero or more path segments in the operator's place; Wildcard path `/*` - matches all paths. Equivalent to `/{**}` path. | Value can contain operators `{*}` and/or `{**}`. Can also be a wildcard match `/*`                                    |

Reviewer: ```suggestion
| **rules.path**                                   | **YES**  | Specifies the path on which the service is exposed. The supported configurations are:<ul><li>Exact path (e.g. `/abc`) - matches the specified path exactly.</li><li>Usage of the `{*}` operator (e.g. `/foo/{*}` or `/foo/{*}/bar`) - matches any request that matches the pattern with exactly one path segment in the operator's place.</li><li>Usage of the `{**}` operator (e.g. `/foo/{**}` or `/foo/{**}/bar`) - matches any request that matches the pattern with zero or more path segments in the operator's place. `{**}` must be the last operator in the path.</li><li>Wildcard path `/*` - matches all paths. It's equivalent to the `/{**}` path.</ul>
| The value might contain operators `{*}` and/or `{**}`. It can also be a wildcard `/*`.                                    |
```
